plugins/pdr_deterministic_plugin/tests/simulation_telemetry.py

```diff
@@ -219,8 +225,29 @@ def start_server(port:str,changes_intervals:int, run_forever:bool):
         if not run_forever and ENDPOINT_CONFIG["ITERATION_TIME"] > MAX_ITERATIONS:
             # after all the tests are done, we need to stop the simulator and check the logs
             return
+        if not setup_log_test:
+            setup_log_test = create_telemetries_logs()
         time.sleep(changes_intervals)
 
+def create_telemetries_logs():
+    """
+    create up to double of the amount of logs that can be in the folder. this is a setup for test.
+    """
+    successful = True
+    for option_location in ['abs','delta']:
+        input_path_dir = Path(join(TELEMETRY_DATASTORE_LOCATION,option_location))
+        files = list(input_path_dir.glob("*.csv"))
+        if len(files)==0:
+            print("didnt create files because there are no files in the folder")
```

Comment:
please change the message to:
"failed to detect files in the folder {input_path_dir}, no files were created"

Reply:
I removed this part and only copied the abs file as it was created way before, and it does not matter which file we copied there it is the same format and has data.